charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
-        {{ if .Values.controllerManager.apiserverSkipVerify -}}
+        {{ if and (.Values.controllerManager.apiserverSkipVerify) (not .Values.useAggregator) -}}
```

Comment:
It should work, but probably the better option would be to not set the `controllerManager.apiserverSkipVerify` flag for aggregated configuration in the first place?